docs/faq.rst

```diff
@@ -3,9 +3,123 @@ FAQ
 
 Here you'll find answers to some frequently asked questions.
 
+Breaking changes in tox 4
+-------------------------
+Version 4 of tox should be mostly backwards compatible with version 3, with the following exceptions:
+
+tox 4 - Python support
+++++++++++++++++++++++
+- Now requires Python ``3.7`` or later and is tested only against CPython. You can still create test environments for
+  earlier Python versions or different Python interpreters. PyPy support is best effort, meaning we do not test it as
+  part of our CI runs, however if you discover issues under PyPy we will accept PRs addressing it.
+
+tox 4 - known regressions
++++++++++++++++++++++++++
+
+- With tox 4 the tty trait of the caller environment is no longer passed through. The most notable impact of this is
+  that some tools no longer print colored output. A PR to address this is welcomed, in the meantime you can use the
+  ``tty`` substitution to force color mode for these tools, see for example tox itself with pytest and mypy
+  `here in tox.ini <https://github.com/tox-dev/tox/blob/main/tox.ini#L28>`_.
+
+tox 4 - new plugin system
++++++++++++++++++++++++++
+
+tox 4 is a grounds up rewrite of the code base, and while we kept the configuration layer compatibility no such effort
+has been made for the programmatic API. Therefore, all plugins will need to redo their integration against the new code
+base. If you're a plugin developer refer to the `plugin documentation <https://tox.wiki/en/latest/plugins.html>`_ for
+more information.
+
+tox 4 - removed tox.ini keys
+++++++++++++++++++++++++++++
+
++--------------------------+---------------------------------------------+
+| Configuration key        | Migration path                              |
++==========================+=============================================+
+| ``indexserver``          | See `Using a custom PyPI server`_.          |
++--------------------------+---------------------------------------------+
+| ``whitelist_externals``  | Use :ref:`allowlist_externals` key instead. |
++--------------------------+---------------------------------------------+
+| ``isolated_build``       | Isolated builds are now always used.        |
++--------------------------+---------------------------------------------+
+
+tox 4 - substitutions removed
++++++++++++++++++++++++++++++
+- The ``distshare`` substitution has been removed.
+
+tox 4 - CLI arguments changed
++++++++++++++++++++++++++++++
+- The ``--parallel--safe-build`` CLI argument has been removed, no longer needed.
+- When you want to pass an option to a test command, e.g. to ``pytest``, now you must use ``--`` as a separator, this
+  worked with version 3 also, but any unknown trailing arguments were automatically passed through, while now this is
+  no longer the case.
+
+tox 4 - packaging changes
++++++++++++++++++++++++++
+- We use isolated builds (always) as specified by :pep:`518` and use :pep:`517` to communicate with the build backend.
```

Comment:
Can you quickly explain to me what `isolated builds` mean?

Do we expect many broken CIs because of that change?

What can people do about it? Would https://pypi.org/project/tox-current-env/ help with their potential issues? Or is this something completely different.

Reply:
Read the linked PEPs to find out. That is all the information you need.